frontend/src/components/App/PreFishbowl/PreFishbowlParticipants.tsx

```diff
@@ -68,6 +54,20 @@ const PreFishbowlParticipants = () => {
   };
 
   useEffect(() => {
+    const pingParticipant = () => {
```

Comment:
Why this function need to be inside useEffect?

Reply:
because it is not used outside the useEffect

Reply:
the eslint sends the error if is not used outside the useEffect?

Reply:
yes. If you use the function on multiple useEffect you can use `useCallback` , you have multiple examples on the PR.